test/test_torch.py

```diff
@@ -14293,15 +14347,23 @@ def test_hardswish(self, device, dtype):
         self.assertEqual(inputTensorCpy, expectedOutputTensor, precision_4dps)
 
     @onlyCPU
-    @dtypes(torch.float, torch.double)
+    @dtypes(torch.float, torch.double, torch.bfloat16)
```

Comment:
you can use @precisionOverride decorators here instead of manually setting precision